.changeset/README.md

```diff
@@ -0,0 +1,8 @@
+# Changesets
+
+Hello and welcome! This folder has been automatically generated by `@changesets/cli`, a build tool that works
+with multi-package repos, or single-package repos to help you version and publish your code. You can
+find the full documentation for it [in our repository](https://github.com/changesets/changesets)
+
+We have a quick list of common questions to get you started engaging with this project in
+[our documentation](https://github.com/changesets/changesets/blob/main/docs/common-questions.md)
```

Comment:
Can we have a section here for beta releases?

```
## Beta releases


npx changeset-recover@beta


edit the generated files to tidy up the generate Changelog entry:
   - select whether the change is a bugfix (patch), minor change, or major / breaking change
   - edit the changelog entry, which defaults to a combination of both the PR title and PR description - and if an associated PR can't be found, the commit message will be used (useful for direct commits to main)
   ```

Reply:
changeset-recover is a fallback tool, for when the users of changeset don't want to use the changeset-recommended workflow. idk that it needs to be mentioned here.

All will be clear when we prepare for a release, I promise 🎉